src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -67,22 +69,28 @@ public void getAccount_whenExist_returnOneAccount() {
     @Test
     public void getAccount_whenNotExist_returnEmptyAccountResponse() {
         // given
-        final var fakeAccountId = fakeDataGenerator.generateRandomInteger();
-        final var emptyAccountResponse = new AccountResponse();
+        final var fakeAccount = fakeDataGenerator.generateFakeAccount();
+        final var fakeAccountId = fakeAccount.getId();
+        final var fakeAccountResponse = fakeDataGenerator.generateFakeAccountResponse();
+        fakeAccountResponse.setId(null);
+        fakeAccountResponse.setActive(null);
+        fakeAccountResponse.setDateCreated(null);
+        fakeAccountResponse.setPets(null);
+        fakeAccountResponse.setAccountHolders(null);
```

Comment:
could we make this a separate method in the FakeDataGenerator?
Something like `generateEmptyAccountResponse`?